src/index.html

```diff
@@ -9,6 +9,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img 
+          src="./images/logo.png" 
+          alt="logo_moyo" 
+          class="logo_pic"
+        >
+      </a>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href=" " class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link is-active">Apple</a>
```
use `#` as default href